onnxruntime/core/flatbuffers/create_python_bindings.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import subprocess
+
+
+def update_namespace():
+    # create a copy of the schema so we can replace the namespace so that the generated module name doesn't clash
+    # with the 'onnxruntime' package.
+    with open('schema/ort.fbs', 'r') as input, open('schema/ort.py.fbs', 'w') as output:
+        for line in input:
+            # convert any line with the namespace to use ort_flatbuffers_py instead of onnxruntime as the top level
+            # namespace. this doesn't change how anything works - it just avoids a naming clash with the 'real'
+            # onnxruntime python package
+            output.write(line.replace('onnxruntime.experimental.fbs', 'ort_flatbuffers_py.experimental.fbs'))
+
+
+def generate_schema(flatc):
+    # run flatc to generate schema
+    cmd = [flatc, '--python', 'schema/ort.py.fbs']
```

Comment:
Do we want to put all these generate python files under onnxruntime/core? Maybe under tools/python?

Reply:
they're under onnxruntime/core/flatbuffers where we have other things generated from the schema. I'd rather keep them close to the schema so it's a bit more obvious there's a dependency there.

Reply:
Could potentially move them down under the schema directory though so it's closer to the generated ort.fbs.h and readme.md.